examples/templates/bare/main.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    coder = {
+      source = "coder/coder"
+    }
+  }
+}
+
+# This example does not provision any resources. Use this
+# template as a starting point for writing custom templates
+# using any Terraform resource/provider.
+#
+# see: https://coder.com/docs/coder-oss/latest/templates
+
+data "coder_workspace" "me" {
+
+}
+
+resource "coder_agent" "dev1" {
+  os   = "linux"
+  arch = "amd64"
+  auth = "token"
+}
+
+resource "null_resource" "fake-compute" {
+  # When a workspace is stopped, this resource is destroyed.
+  count = data.coder_workspace.me.transition == "start" ? 1 : 0
+
+  provisioner "local-exec" {
+    command = "echo 🔊 ${data.coder_workspace.me.owner} has started a workspace named ${data.coder_workspace.me.name}"
+  }
+
+  # Run the Coder agent init script on resources
+  # to access web apps and SSH:
+  #
+  # ${coder_agent.dev1.init_script}
```

Comment:
should probably also mention `coder_agent.dev1.token`

Reply:
great point!